templates/auth/register.html

```diff
@@ -48,6 +48,11 @@ <h1 class="card-title">Create new account</h1>
       <div class="mt-2">
         <button type="submit" class="btn btn-primary btn-block">Create Account</button>
       </div>
+
+      {% if connect_with_proton %}
+        <div class="text-center my-2 text-gray"><span>or</span></div>
+        <a class="btn btn-primary btn-block mt-2 proton-button" href="{{ url_for("auth.proton_login") }}">Sign up with Proton</a>
+      {% endif %}
```

Comment:
In login.html, it seems that we need to move this button out of the <form> but we don't need to do it here, is this normal?

Reply:
If a user doesn't have a SL account but does have a proton one, they may think that they need to create an account first. We could also replace the button for a small text with an explanation like 'If you have a proton account just log in'. Or do would you prefer to remove this entirely?

Reply:
I think we don't need to mention this to not overcharge the UI and both buttons have the same logic behind anw.